lib/modules/apostrophe-schemas/index.js

```diff
@@ -1395,6 +1395,29 @@ module.exports = {
       }
     });
 
+    self.addFieldType({
+      name: 'colorpicker',
+      converters: {
+        string: function (req, data, name, object, field, callback) {
```

Comment:
This must actually sanitize. Make sure it's a string (self.apos.launder.string), then make sure it's in the format you accept.

It should be possible for a color to be left null (a non-required field), so if the string is empty that should be treated as null, and on the populate/editing end of things there should be a way to go back to having no selection.